spk/sickchill/Makefile

```diff
@@ -1,23 +1,23 @@
 SPK_NAME = sickchill
 SPK_VERS = 20210729
-SPK_REV = 2
+SPK_REV = 3
 SPK_ICON = src/sickchill.png
 
-DEPENDS = cross/$(SPK_NAME)
-SPK_DEPENDS = "python38:git"
-PIP=pip3
+BUILD_DEPENDS = cross/python38 cross/poetry cross/lxml
+# cross/cffi cross/setuptools cross/cryptography
+DEPENDS = cross/PyNaCl cross/$(SPK_NAME)
+SPK_DEPENDS = "python38"
 
-WHEELS = src/requirements.txt
+WHEELS = src/requirements-cross.txt src/requirements.txt
 
 MAINTAINER = miigotu
 DESCRIPTION = Automatic Video Library Manager for TV Shows. It watches for new episodes of your favorite shows, and when they are posted it does its magic.
-DESCRIPTION_SPN = Gestor automático para bibliotecas de series. Busca nuevos episodios de tus series favoritas, y cuando son publicados hace su magia.
```

Comment:
Is the Spanish wrong? Why was the translation removed?

Reply:
Nothing else is in Spanish, and then what about other languages too. 
This simplified everything to one language.

Reply:
We do have Spanish wizard files for sickchill here: https://github.com/SynoCommunity/spksrc/tree/master/spk/sickchill/src/wizard. It's true the vast majority is only in English and only few have French translations however I think it’s nice to keep as much support for multilingual packages as we can.